.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "env": {
+    "cjs": {
+      "presets": ["es2015", "react", "stage-2"],
```

Comment:
there is no JSX code, and aside from `import` modern browsers, as well as the current LTS of Node itself supports ES6, so I see no reason to do all these things. Just rollup, set to rollup to UMD, should be enough. 

See https://github.com/rollup/rollup/wiki/JavaScript-API but then with the obvious `format: 'umd'`